apis/python/examples/normalizing.md

```diff
@@ -0,0 +1,35 @@
+## Normalizing a SOMA collection
+
+The [cartographer script](../examples/cartographer.py) shows an example of how to take a collection
+of H5AD files -- and/or already-ingested SOMAs -- and _normalize_ them into a uniform collection.
```

Comment:
"... and aggregate them into a uniform collection." (again, not overusing the world normalize)

Reply:
@bkmartinjr thanks! i want to be clear we're not doing any `concat` operation though ... if someone has 27 `.h5ad` files they'll have a SOCO with 27 SOMAs in it ... how best clearly to indicate that fact?